jujugui/static/gui/src/app/app.js

```diff
@@ -801,7 +801,8 @@ YUI.add('juju-gui', function(Y) {
           instanceName="white-box"
           visible={visible}>
           <components.SearchResults
-            query={text} />
+            query={text}
+            charmstore={this.get('charmstore')} />
```

Comment:
Not sure if we want to pass the charmstore into the component here, or just the search function. @hatched might have an opinion here.